mr/local_runner.cc

```diff
@@ -351,6 +352,21 @@ void LocalRunner::Impl::End(ShardFileMap* out_files) {
   }
   dest_mgr_->CloseAllHandles(stop_signal_.load(std::memory_order_acquire));
 
+  string metric_file =
```

Comment:
The task was to unify the counters in operation executors to a single data structure and to save it. I understand that this PR handles only saving.
I think it's better if we save once after the pipeline run.... not after each operator run

Reply:
Fixed and added support for writing through GS.